File: C#/4. PizzaStore/Classes/Chicago Pizza Store/ChicagoStyleCheesePizza.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PizzaStore.Classes.Chicago_Pizza_Store
{
    public class ChicagoStyleCheesePizza : Pizza
    {
        public ChicagoStyleCheesePizza()
        {
            _Name = "Chicago Style Deep Dish Cheeese Pizza";
            _Dough = "Extra Thick Crust Dough";
            _Sauce = "Plum Tomato Sauce";

            _Toppings.Add("Shredded Mozzarella Cheese");
        }

        public override void Cut()
        {
            Console.WriteLine("Cutting the pizza into square slices");
        }
    }
}

File: Java/4-PizzaStoreFactory/src/main/java/com/mycompany/pizzastorefactory/ChicagoStylePizza/ChicagoStyleCheesePizza.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.pizzastorefactory.ChicagoStylePizza;

import com.mycompany.pizzastorefactory.Pizza;

/**
 *
 * @author ismas
 */
public class ChicagoStyleCheesePizza extends Pizza {
    public ChicagoStyleCheesePizza(){
        name = "Chicago Style Deep Dish Cheese Pizza";
        dough = "Extra Thick Crust Dough";
        sauce = "Plum Tomato Sauce";
        
        toppings.add("Shredded Mozzarela Cheese");
    }
    
    @Override
    protected void cut(){
        System.out.println("Cutting the pizza into square slices");
    }
}
